content/integrations/dnscheck.md

```diff
@@ -36,7 +39,16 @@ To validate that the DNS Service Check is working, run `datadog-agent info`. You
       ---------
         - instance #0 [OK]
         - Collected 1 metric, 0 events & 2 service checks
+{{< /highlight >}}
 
 ## Usage
 
-When the data has been submitted to Datadog, you will have one new metric: `dns.response_time`. This will have the tags `nameserver:<nameserver_in_yaml_file>` and `resolved_hostname:<hostname_in_yaml>`.
+Data submitted to Datadog is tagged with the tags `nameserver:<nameserver_in_yaml_file>` and `resolved_hostname:<hostname_in_yaml>`. 
```

Comment:
This check tags everything it collects with:

* `nameserver:<nameserver_in_yaml>`
* `resolved_hostname:<hostname_in_yaml>`